README.md

```diff
@@ -41,7 +41,8 @@ To start working on the Agent, you can build the `master` branch:
 2. cd into the project folder: `cd $GOPATH/src/github.com/DataDog/datadog-agent`.
 3. install project's dependencies: `invoke deps`.
    Make sure that `$GOPATH/bin` is in your `$PATH` otherwise this step might fail.
-4. build the whole project with `invoke agent.build --build-exclude=systemd` (with `--use-venv` to use a python virtualenv)
+4. Build the `six` dependency with `invoke six.build && inv six.install`. You will need CMake installed and a C++ compiler for this to work.
+5. build the agent with `invoke agent.build --build-exclude=systemd` (with `--use-venv` to use a python virtualenv)
```

Comment:
`use-env` is no longer used, we could remove it from our tasks. You need to use `python-home-2` and `python-home-3`